TheFourFifths/consus-client
# Overdue items tff 78

### Summary

Removed ability to check out an item when the student in question has an overdue item. Also added "timestamp" field to items that a student has checked out. This is the time it is due back. Another change that was made as that the student now comes with the items they have checked out instead of just their addresses. This makes life easier client-side. 

NOTE: This PR has (or will have if you read this early enough) a sister PR on the 'consus' repo that will contain the backend changes required for this. 

### Checklist


Finally, is it ready to be reviewed and merged: :white_check_mark:

### How to Review

1. Ensure modified tests still make sense.
2. Ensure that any changes made to the stores don't conflict with proper working order.

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-78](https://msoese.atlassian.net/browse/TFF-78)
